src/pip/_internal/vcs/subversion.py

```diff
@@ -37,7 +54,57 @@ def should_add_vcs_url_prefix(cls, remote_url):
     def get_base_rev_args(rev):
         return ['-r', rev]
 
+    def vcs_global_options(self):
+        # type: () -> List[str]
+        """Options to be used on calls to Subversion that contact the server.
+
+        These options are applicable for the following ``svn`` subcommands used
+        in this class.
+
+            - checkout
+            - export
+            - info
+            - switch
+            - update
+
+        :return: A list of command line arguments to pass to ``svn``.
+        """
+        options = []  # type: List[str]
```

Comment:
I think this function will be simpler without this variable. You can just return the list instead of appending to an empty list.